evap/results/exporters.py

```diff
@@ -120,6 +120,14 @@ def export(self, response, course_types_list, include_not_enough_voters=False, i
             for evaluation, results in evaluations_with_results:
                 writec(self, evaluation.full_name, "evaluation")
 
+            writen(self, _("Course Degree"), "bold")
+            for evaluation, results in evaluations_with_results:
+                writec(self, "\n".join([d.name for d in evaluation.course.degrees.all()]), "border_left_right")
```

Comment:
please use a style `degree` here (instead of `border_left_right`) and define it above (line 47) as follows:
```
'degree':           xlwt.easyxf('alignment: wrap on; borders: left medium, right medium'),
```